apps/frontend/src/app/5_pages/ConvertPage/ConvertPage.tsx

```diff
@@ -78,10 +83,22 @@ const ConvertPage: FC = () => {
 
   const [quote, setQuote] = useState('');
   const [route, setRoute] = useState<SwapRoute | undefined>();
+  const defaultSourceToken = useMemo(() => {
+    if (
+      fromToken &&
+      fromToken in SupportedTokens &&
+      fromToken !== SupportedTokens.fish
+    ) {
+      return fromToken as SupportedTokens;
+    }
+    if (fromToken === SupportedTokens.fish) {
+      return SupportedTokens.fish;
+    }
```

Comment:
I think you wanted to rewrite `fish` to `babelfish` here?

```suggestion
    if (fromToken === SupportedTokens.fish) {
      return SupportedTokens.babelfish;
    }
```

Reply:
if so, rewrite `fish` to `babelfish` for `to` field too, if `fish` comes in as a query param

Reply:
@creed-victor but we don't have a `babelfish` in SupportedTokens, it's a value, and key is a `fish`

Reply:
ah right, then maybe it was supposed to be:
```suggestion
    if (fromToken === "fish") {
      return SupportedTokens.fish;
    }
```
@pietro-maximoff

Reply:
@creed-victor it won't work that way, because when we select the `fish` token the value will be `babelfish` and this condition `(fromToken === "fish")` won't work